code/controllers/master.dm

```diff
@@ -127,6 +128,12 @@ var/global/datum/controller/master/Master = new()
 
 				var/ran_subsystems = 0
 				for(var/datum/subsystem/SS in subsystems)
+					if(world.cpu >= 100)
+						//if world.cpu gets above 120,
+						//byond will pause most client updates for (about) 1.6 seconds.
+						//(1.6 seconds worth of ticks)
+						//We just stop running subsystems to avoid that.
+						break;
```

Comment:
>\>using a semicolon as an eol

How sad is it that the best tool we have to avoid lag is to lag things up manually? Either way :+1: looks ok